lib/LedgerSMB/Report/Unapproved/Batch_Detail.pm

```diff
@@ -109,12 +109,14 @@ sub columns {
        type => 'text',
      pwidth => 2, },
 
-    {col_id => 'default_date',
+    # Correct variable name for date is transaction_date
+    # default_date doesn't exist
```

Comment:
This makes for a nice comment in the commit message, but not so much as a source comment: after the change goes in, nobody looks at the diff. Then, when you open the file, you see "default_date doesn't exist". Sure, but there's nothing actually talking about "default_date" in the sources at that point, right?

Reply:
Your comment make sense. Should I move these comment to a commit msg and synchronize this PR or should I create a new clean PR?

Reply:
Edit and resync is totally fine by me.

Reply:
new commit was pushed for the changes. Please review. Please feel free to merge if it is okay.